.gitignore

```diff
@@ -1,78 +0,0 @@
-# based on https://github.com/github/gitignore/blob/master/Python.gitignore
```

Comment:
something went wrong here, you deleted the entire gitignore

Reply:
I was under the impression that the .gitignore was from transferred from my branch.. because we were getting rid of that yesterday in my branch and master.

Reply:
Its fixed now and .gitignore is now recovered again. Thanks alot @tepals !